spec/support/capybara.rb

```diff
@@ -0,0 +1,28 @@
+require 'capybara-screenshot/rspec'
+require 'selenium/webdriver'
+
+Capybara.javascript_driver = :headless_chrome
+
+Capybara.configure do |config|
+  config.default_driver = :headless_chrome
+  config.javascript_driver = :headless_chrome
+
```

Comment:
Do we need both the js_driver configs, in Line 4 _and_ line 8?

Reply:
Probably not - thanx for pointing out 🙏